conf/app.conf

```diff
@@ -0,0 +1,3 @@
+appname = go-google-scraper-challenge
+httpport = 8080
```

Comment:
We can add `httpaddr` for host also

Reply:
@junan ✅  added in [e4c6f66](https://github.com/carryall/go-google-scraper-challenge/pull/16/commits/e4c6f662968fd367976afa969a1fe4cf2d06deb4)